### PR TITLE
Add faust module

### PR DESCRIPTION
**Pull request by magnetophon**
I'm just looking for a catchy description for in  init.example.el. ;)

### CODE REVIEW

File: modules/lang/faust/packages.el [hunk 1]
```diff
@@ -0,0 +1,7 @@
+;; -*- no-byte-compile: t; -*-
+;;; lang/faust/packages.el
+
+(package! faustine)
+(package! projectile)
+(package! auto-complete)
+(package! yanippet)
```
Comment (hlissner) [inline modules/lang/faust/packages.el]: None of these packages are required. 

1. Leave it to the user to enable the `:editor snippets` module to provide yasnippet. 
2. Projectile is provided by Doom core.
3. I'm going to replace its dependence on auto-complete with company.

File: modules/lang/faust/README.org [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
++ Fully configurable (build type/target/architecture/toolkit, keyboard shortcuts, etc.)
++ Automatic keyword completion (if Auto-Complete is installed)
++ Automatic objets (functions, operators, etc.) template insertion with default sensible values (if Yasnippet is installed)
++ Modeline indicator of the state of the code
+
+** Plugins
++ [[https://bitbucket.org/yphil/faustine][Faustine]]
++ [[https://github.com/bbatsov/projectile][Projectile]]
++ [[https://github.com/auto-complete/auto-complete][AutoComplete]]
++ [[https://github.com/joaotavora/yasnippet][Yasnippet]]
```
Comment (hlissner) [inline modules/lang/faust/README.org]: The description of each label should match the plugin's name. `Faustine` -> `faustine`, `AutoComplete` -> `auto-complete`, etc.

Also, please remove projectile, autocomplete and yasnippet (see my review for packages.el).

File: modules/lang/faust/README.org [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#+TITLE:   lang/faust
+#+DATE:    July 23, 2019
+#+SINCE:   {replace with next tagged release version}
```
Comment (hlissner) [inline modules/lang/faust/README.org]: Use `v2.1.0`

**Comment by magnetophon** [comment]
Thanks for the quick review!
All done.

**Comment by magnetophon** [comment]
Thanks!

**Pull request merged**